Managed Services/cloudera-add-on-services-available-on-centurylink-cloud.md

```diff
@@ -0,0 +1,45 @@
+</p>
+<p><strong>Key-Value Store Indexer (Requires HBase and Solr)</strong></p>
+<p>The Key-Value Store Indexer service uses the Lily HBase NRT Indexer to index the stream of records being added to HBase tables. Indexing allows you to query data stored in HBase with the Solr service.
+</p>
+<p><strong>Navigator</strong></p>
+<p>Cloudera Navigator is the only native end-to-end governance solution for Apache Hadoop-based systems. Through a single user interface, it provides visibility for administrators, data managers, data scientists, and analysts to secure, govern, and explore the large amounts of diverse data that land in Hadoop. Cloudera Navigator is part of Cloudera Enterprise’s comprehensive data security and governance offering and is a key part to meeting compliance and regulatory requirements.
+<br>
+<u>[More Information on Navigator](http://www.cloudera.com/content/cloudera/en/products-and-services/cloudera-enterprise/cloudera-navigator.html)</u>
+</p>
+<p><strong>Sentry</strong></p>
+<p>A key part of compliance-ready security is controlling access to data. Apache Sentry (incubating) is a unified authorization mechanism so you can store sensitive data in Hadoop. Sentry is a fully integrated component of CDH and provides fine-grained authorization and role-based access control all through a single system. Sentry currently integrates with the open source SQL query frameworks, Apache Hive and Cloudera Impala, and the open source search engine, Cloudera Search, and can also extend to other computing engines within the Hadoop ecosystem. Sentry is a crucial part of compliance-ready security.
+<br>
+<u>[More Information on Sentry](http://www.cloudera.com/content/cloudera/en/products-and-services/cdh/sentry.html)</u>
+</p>
+<p><strong>Solr</strong></p>
+<p>Solr is a standalone enterprise search server with a REST-like API. You put documents in it (called "indexing") via JSON, XML, CSV or binary over HTTP. You query it via HTTP GET and receive JSON, XML, CSV or binary results. Solr is highly reliable, scalable and fault tolerant, providing distributed indexing, replication and load-balanced querying, automated failover and recovery, centralized configuration and more. Solr powers the search and navigation features of many of the world's largest internet sites.
+<br>
+<u>[More Information on Solr](http://lucene.apache.org/solr)</u>
+</p>
+<p><strong>Spark</strong></p>
+<p>Apache Spark is an open source, parallel data processing framework that complements Apache Hadoop to make it easy to develop fast, unified Big Data applications combining batch, streaming, and interactive analytics on all your data. For analysts and data scientists who rely on iterative algorithms (e.g. clustering/classification), Spark is 10-100x faster than MapReduce delivering faster time to insight on more data, resulting in better business decisions and user outcomes.
+<br>
```

Comment:
`br` tags should not be used. and if they are used they need to be self closing `<br/>`